### PR TITLE
Accept applicants

### LINKED ISSUE #48: Accept / reject attendees
Implemented as part of the admin panel. Needs to send out emails and allow access to a second evaluation form.

### PR DESCRIPTION
**Pull request by petschekr**
Adds a section to the admin panel for accepting users.

Closes #48

### CODE REVIEW

File: client/css/admin.css [hunk 1]
```diff
@@ -18,8 +18,20 @@ table th, table td {
     padding: 12px 4px;
 }
 table td:last-child {
-    text-align: center;
     width: 50px;
+    text-align: center;
+}
+#applicants table td:last-child {
+    width: 140px;
```
Comment (BunsenMcDubbs) [inline client/css/admin.css]: The used of `px` in principle triggers me. But it 200% doesn't matter. Just my 2 cents.

File: server/routes/templates.ts [hunk 2]
```diff
@@ -234,10 +234,13 @@ templateRoutes.route("/admin").get(authenticateWithRedirect, async (request, res
 	if (!user.admin) {
 		response.redirect("/");
 	}
+	let rawQuestions = await validateSchema("./config/questions.json", "./config/questions.schema.json");
```
Comment (BunsenMcDubbs) [inline server/routes/templates.ts]: This only runs once right?

Comment (petschekr) [inline server/routes/templates.ts]: It'll run once whenever `/admin` is requested. Is that what you meant?

Comment (BunsenMcDubbs) [inline server/routes/templates.ts]: Oh whoops, my bad. 👍

**Pull request merged**